ground_truth_labeling_jobs/video_annotations_quality_assessment/Ground_Truth_Video_Quality_Metrics.ipynb

```diff
@@ -8,18 +8,9 @@
     "\n",
```

Comment:
<p>Should we link to the Object tracking specific part of the output data docs in this link at the end? https://docs.aws.amazon.com/sagemaker/latest/dg/sms-data-output.html#sms-output-video-object-tracking </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aws/amazon-sagemaker-examples/pull/2164/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='18'/>